src/js/jsx/sections/nodoc/RecentFiles.jsx

```diff
@@ -72,17 +73,25 @@ define(function (require, exports, module) {
                         </li>
                     );
                 }, this);
-
-            return (
-                <section className="recent-files section">
-                    <TitleHeader title={strings.NO_DOC.RECENT_FILES_TITLE} />
-                    <div className="section-container">
-                        <ul className="link-list__list">
-                            {recentFilelinks}
-                        </ul>
-                    </div>
-                </section>
-            );
+                
+            if (recentFilesCount >= 1) {
```

Comment:
And, if you gotta dive back in anyway, please change this to use a strict inequality for better consistency with our style `recentFilesCount > 0`. Alternatively, you can eliminate the count variable and just use the `isEmpty()` method. http://facebook.github.io/immutable-js/docs/#/List/isEmpty